### PR TITLE
Add time & driveaudit options in hummingbird recon middleware

### LINKED ISSUE #28: fix swift-recon --all
right now you get a lot of:

[2017-03-21 23:48:41] Validating server type 'object' on 4 hosts...
-> http://127.0.0.1:6030/: HTTP Error 405: Method Not Allowed
-> http://127.0.0.1:6040/: HTTP Error 405: Method Not Allowed
-> http://127.0.0.1:6010/: HTTP Error 405: Method Not Allowed
-> http://127.0.0.1:6020/: HTTP Error 405: Method Not Allowed
4/4 hosts ok, 0 error[s] while checking hosts.
===============================================================================
[2017-03-21 23:48:41] Checking drive-audit errors
-> http://127.0.0.1:6030/recon/driveaudit: HTTP Error 404: Not Found
-> http://127.0.0.1:6040/recon/driveaudit: HTTP Error 404: Not Found
-> http://127.0.0.1:6010/recon/driveaudit: HTTP Error 404: Not Found
-> http://127.0.0.1:6020/recon/driveaudit: HTTP Error 404: Not Found
[drive_audit_errors] - No hosts returned valid data.

### PR DESCRIPTION
**Pull request by nadeemsyed**
Also add support for OPTIONS call in object & container server.

Fixes #28

### CODE REVIEW

File: middleware/recon.go [hunk 1]
```diff
@@ -427,6 +427,16 @@ func ReconHandler(driveRoot string, writer http.ResponseWriter, request *http.Re
 			http.Error(writer, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
 			return
 		}
+	case "time":
+		//Similar to python time.time()
+		content = float64(time.Now().UnixNano()) / 1000000000
```
Comment (dpgoetz) [inline middleware/recon.go]: time.Second instead of the 100000000000000

File: common/version.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
+//  Copyright (c) 2015 Rackspace
```
Comment (dpgoetz) [inline common/version.go]: i guess change to 2017 since there's already the other fix

File: middleware/recon.go (hunk 1 above)
Comment (nadeemsyed) [inline middleware/recon.go]: Done.

File: common/version.go (hunk 2 above)
Comment (nadeemsyed) [inline common/version.go]: Done.

**Pull request merged**